apollo-compiler/src/test/typename/fragment_spread.graphql

```diff
@@ -0,0 +1,13 @@
+query TestQuery {
+    hero {
+        ...heroFragment
+    }
+}
+
+fragment heroFragment on Character {
+    # No typename should be appended here
```

Comment:
it should because we have nested inline fragment here `Droid`

Reply:
I thought that as well but since the Fragment spread will always be queried on a parent field, the `__typename` will be added there. In that case, the transformed definitions will be:

```graphql
query TestQuery {
    hero {
        __typename
        ...heroFragment
    }
}

# Fragment did'nt change
fragment heroFragment on Character {
    name
    ... on Droid {
        primaryFunction
    }
} 
```